plugins/logitech-hidpp/fu-logitech-hidpp-runtime.c

```diff
@@ -230,10 +230,10 @@ fu_logitech_hidpp_runtime_setup_internal (FuDevice *device, GError **error)
 		    (self->version_bl_major == 0x03 && config[8] >= 0x02)) {
 			self->signed_firmware = TRUE;
 			fu_device_set_protocol (device, "com.logitech.unifyingsigned");
-		} else {
-			fu_device_set_protocol (device, "com.logitech.unifying");
 		}
 	}
+	if (!self->signed_firmware)
+		fu_device_set_protocol (device, "com.logitech.unifying");
```

Comment:
I'm wondering if we should set com.logitech.unifyingsigned here, but not a hill I want to die on.

Reply:
I think if the code refactors that it makes sense such a second set of circumstances for signed to be set, otherwise not necessary.